backend/src/main/java/com/bakdata/conquery/models/events/stores/specific/DateRangeTypeCompound.java

```diff
@@ -0,0 +1,126 @@
+package com.bakdata.conquery.models.events.stores.specific;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import com.bakdata.conquery.io.cps.CPSType;
+import com.bakdata.conquery.models.common.daterange.CDateRange;
+import com.bakdata.conquery.models.events.Bucket;
+import com.bakdata.conquery.models.events.stores.primitive.IntegerDateStore;
+import com.bakdata.conquery.models.events.stores.root.ColumnStore;
+import com.bakdata.conquery.models.events.stores.root.DateRangeStore;
+import com.bakdata.conquery.models.events.stores.root.DateStore;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * Stores {@link CDateRange} as pair of two {@link IntegerDateStore}s.
+ */
+@CPSType(base = ColumnStore.class, id = "DATE_RANGE_COMPOUND")
+@Getter
+@Setter
+@ToString(of = {"startStore", "endStore"})
+public class DateRangeTypeCompound implements DateRangeStore {
+
+
+	@NotNull
+	@NotEmpty
+	private String startColumn, endColumn;
+
+	@JsonIgnore
+	private DateStore startStore;
+
+	@JsonIgnore
+	private DateStore endStore;
+
+
+	@Setter(AccessLevel.PROTECTED)
+	@JsonIgnore
+	private Bucket parent; // this parent will be set after all the stuff has been preprocessed
```

Comment:
Das in einen Doc-Comment drüber und sauberer schreiben. Der Bucket wird gesetzt nachdem er deserialisiert wird, nicht "stuff" ;)